markdown/dev/tutorials/getting-started-vercel/en.md

```diff
@@ -0,0 +1,221 @@
+(Because of this there are no retention periods or simple ways to
+delete multiple deployments all at once.)
+
+</Tip>
+
+## Creating an account
+
+You will need to sign up for a Vercel account.
+Luckily, you can simply sign up using your GitHub
+account/credentials without having to create a separate username
+or password.
+
+1. On the [Vercel website][v] select the "Sign Up" button.
+(A shortcut URL is: [https://vercel.com/signup][vsu].)
+2. Select the "Continue with GitHub" button.
+3. A pop-up window will appear asking you for permission to access
+your GitHub information.
+Press the green "Authorize Vercel" button to continue.
+
+[v]: https://vercel.com
+[vsu]: https://vercel.com/signup
+
+## Importing a repository
+
+After you create your account you will then see an "Import Git Repository"
+screen.
+The default listed is likely to be the actual `freesewing/freesewing`
+repository owned by FreeSewing.
+However, you should instead import your own personal fork of the
+FreeSewing repository.
+
+1. In the dropdown menu, select "Add a GitHub Account".
+2. Select your personal GitHub acccount from the list.
+3. Select the "Only select repositories" radio button.
+4. In "Select repositories" drop-down menu, select your `freesewing`
+repository.
+5. Click the green "Install" button.
+6. Confirm that you are giving permission to access the repository
+by entering your GitHub password.
+7. Finally, back at the Import Git Repository screen complete the
+import by selecting the white "Import" button.
+
+## Creating a project
+
+You will next be taken to the Configure Project page where you can
+create a project.
+
+By default, the default Root Directory will be `sites/dev`.
+The Root Directory setting will determine the build type for the project.
+- `sites/dev` will build a freesewing.dev website
+- `sites/org` will build a freesewing.org website
+- `sites/lab` will build a lab.freesewing.dev website/app
+
+1. Change the name of the project, if you wish.
+Names can consist of alphanumeric lowercase and hyphen characters.
+2. Change the Root Directory to the desired setting, as described above.
+3. Don't change any of the Build & Development Settings.
+The default values will work fine.
+4. Press white "Deploy" button.
+
+Vercel will then create the project and start building the project's first
+deployment based on the current `develop` branch.
+Once the build completes (in about 3-4 minutes or so)
+you will see a Congratulations page, with a preview image of the
+website home page.
+
+## About deployments
+
+As mentioned previously, Vercel will eventually create and build many
+deployments for your project.
+One is known as the _production deployment_, based on the `develop` branch
+of the repository.
+All other deployments are referred to as _preview deployments_.
+
+Created deployments include:
+1. The initial production deployment for the `develop` branch
+2. A new preview deployment every time you update your `develop` branch in GitHub
```

Comment:
Is this not a new production deployment? If not, how do you get a new production deployment?

Reply:
Added clarification about the production deployment, how it uses the `main` branch, and how it is initially built from `develop` because there is no `main` branch.